### DISCUSSION TITLE
保存笔记到daily notes里面时灵时不灵

### DISCUSSION THREAD

**Discussion by lalaniu85** [zhaohongxuan/obsidian-weread-plugin#77] [Q&A]
不是每次都能和daily notes同步。
而且希望能开放daily notes同步笔记的模版，现有模版和我自己的模版不匹配，做不到引用。
我今天想截图块引用，可是ta怎么都出不来

**Comment by zhaohongxuan** [comment]
Daily note只是一些引用，没有模板，不想做的太重，很多人应该不需要这个功能。

**Reply by lalaniu85** [reply]
好吧

**Reply by Salad-oo** [reply]
项目中有同名文件会出现此问题
